README.md

```diff
@@ -13,15 +13,11 @@ $ npm install live-storage
 ### Create storage
 
 ```js
-import { LiveStorage } from 'live-storage'
+import { LiveStorage } from "live-storage";
 
-const posts = new LiveStorage([])
+const posts = new LiveStorage("posts", []);
 
-posts.update((prevPosts) => {
-  prevPosts.concat({
-    title: 'Brave new world',
-  })
-})
+posts.update((prevPosts) => [...prevPosts, { title: "Brave new world" }]);
```

Comment:
Nitpick: could we use `.concat` and just return its result?

```js
return prevPosts.concat({ ... })
```

That's the same logic that with spreading, yet less foreign syntax, so higher recognizability.

Reply:
will do. even if I would think that spread might be more familiar to beginners these days, concat is definitely more searchable, so probably a better choice for a readme